sw/host/opentitantool/src/command/lc.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use clap::{Args, Subcommand};
+use humantime::parse_duration;
+use serde_annotate::Annotate;
+use std::any::Any;
+use std::time::Duration;
+
+use opentitanlib::app::command::CommandDispatch;
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::dif::lc_ctrl::{DifLcCtrlState, LcCtrlReg};
+use opentitanlib::io::jtag::{JtagParams, JtagTap};
+
+#[derive(serde::Serialize)]
+pub struct LcStateReadResult {
+    pub lc_state: DifLcCtrlState,
+}
+
+#[derive(Debug, Args)]
+/// Reads the device life cycle state over JTAG.
+pub struct LcStateRead {
+    /// Whether or not to use the externally provided clock.
+    #[arg(long)]
+    pub use_ext_clk: bool,
```

Comment:
This doesn't seem to be used?

Reply:
fixed in follow on (https://github.com/lowRISC/opentitan/pull/20288).